ibis/bigquery/compiler.py

```diff
@@ -260,6 +245,13 @@ class BigQueryExprTranslator(impala_compiler.ImpalaExprTranslator):
     _registry = _operation_registry
     _rewrites = impala_compiler.ImpalaExprTranslator._rewrites.copy()
 
+    context_class = BigQueryContext
```

Comment:
This is inconsistent with the `BigQuerySelect.translator` which is not postfixed with `_class`

Reply:
There's `context_class` which is a class variable, and `context` which is an instance of `context_class`, so they are consistent, unless I'm misunderstanding you here.

Reply:
I mean `BigQuerySelect.translator` is holding a class too `BigQueryExprTranslator` not an instance of it.

Reply:
I'll make a follow up issue to fix the inconsistency. In the meantime, I'm going to merge this next.

Reply:
Great! :) You might check https://github.com/ibis-project/ibis/pull/1315 next